pkg/apis/core/v1/validation/validation.go

```diff
@@ -137,6 +137,16 @@ func ValidatePodLogOptions(opts *v1.PodLogOptions) field.ErrorList {
 		if *opts.SinceSeconds < 1 {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("sinceSeconds"), *opts.SinceSeconds, "must be greater than 0"))
 		}
+	case opts.UntilSeconds != nil && opts.UntilTime != nil:
```

Comment:
Making this part of the same switch as `Since*` means that only one of them will be validated.  You want a new switch.

simplified demonstration:
https://play.golang.org/p/zDstx129jMe

Please add a test case that covers that.

Reply:
I modelled this againso you can see:

https://play.golang.org/p/0hJDEO6S61U

I expect both field pairs to be validated, even if one of them fails.